pkg/render/common/meta/meta.go

```diff
@@ -43,9 +43,15 @@ const (
 
 var (
 	// TolerateMaster allows pod to be scheduled on master nodes
-	TolerateMaster = corev1.Toleration{
-		Key:    "node-role.kubernetes.io/master",
-		Effect: corev1.TaintEffectNoSchedule,
+	TolerateMaster = []corev1.Toleration{
```

Comment:
While we're here we should really rename this from "master" to "controlplane" throughout.